lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2727,6 +2727,9 @@ def print_pdf(self, filename, *,
             else:            # we opened the file above; now finish it off
                 file.close()
 
+    def draw(self):
+        _no_output_draw(self.figure)
+        return super().draw()
```

Comment:
missing newline; also for backend_svg

Reply:
Why isn't flake8 dinging me for this?

Reply:
because we explicitly disable E305 due to the use of patterns like
```
def foo():
    ...
foo.__doc__ = ...
```
where putting two extra blank lines would clearly make things worse.